users/style.css

```diff
@@ -145,10 +149,21 @@ li p {
   margin: 150px auto;
 }
 
+#loader_tag {
+  border: 5px solid var(--gray-color);
+  border-top: 5px solid var(--loader-blue-color);
+  border-radius: 50%;
+  width: 50px;
+  height: 50px;
+  animation: spin 2s linear infinite;
+  margin: 150px auto;
+}
+
 .pagination {
   display: flex;
   justify-content: space-between;
   margin-top: auto;
+  visibility: hidden;
```

Comment:
You know that this element will still take the space if we use visibility hidden? Is it the use case? can you explain? Why not display: none

Reply:
did this so other features that might depend on it are not broken and i can develop the required feature and there were some services that depend on it